corpus/stats.py

```diff
@@ -32,18 +40,46 @@ def tasks(self):
     def run(self):
         with uopen(self.bliss_lexicon, "rt", encoding="utf-8") as f:
             tree = ET.parse(f)
-            iv_words = {
-                orth.text.upper() for orth in tree.findall(".//lemma/orth") if orth.text
-            }
+            if self.casing == "upper":
+                iv_words = {
+                    orth.text.upper()
+                    for orth in tree.findall(".//lemma/orth")
+                    if orth.text
+                }
+            elif self.casing == "lower":
+                iv_words = {
+                    orth.text.lower()
+                    for orth in tree.findall(".//lemma/orth")
+                    if orth.text
+                }
+            else:
```

Comment:
maybe check for `None` (or do you mean `"none"`?) explicitly and add an assertion that the user does not request an unknown casing mode.